src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +59,13 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString() {
```

Comment:
Correct implementation of the interface. Good use of the constant for the prefix.
Also, good attempt on handling privacy setting!